src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,5 @@
+.catalog {
+  display: flex;
+  width: 1024px;
+  margin: 0 auto;
+}
```

Comment:
try adjusting style a bit to pass the last test - your entire component should be 1200px wide on respective screen
```
.catalog {
    display: flex;
    min-width: 1024px;
    justify-content: center;
}
```